factory-containers/assemble-system-image

```diff
@@ -150,6 +151,35 @@ def dump_apps_container_images(target: dict, containers_arch: str, app_image_tar
                                    '-C', os.path.join(tmp_docker_dir, containers_arch), '.'])
 
 
+def resize_wic(wic_image: str, increase_bytes: int):
+    increase_k = ceil(increase_bytes / 1024) + 1
+    wic_k = int(os.stat(wic_image).st_size / 1024)
```

Comment:
wic_k will be rounded down, but increase_k will be rounded up... this *might* be ok?  Do we want to round both up?

Reply:
i'll round up. i *think* that value is always a multiple of 1024, but your reasoning makes sense